### ISSUE TITLE
aria-keyshortcuts may be misleading

### ISSUE THREAD

**Issue by babielmam** [Leaflet/Leaflet#10095]
### Checklist

- [x] I've searched through the [plugins](https://leafletjs.com/plugins.html) to make sure this feature isn't already available, or think it shouldn't require a plugin.
- [x] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this feature hasn't been requested already.
- [x] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Motivation

In https://github.com/Leaflet/Leaflet/pull/9688 @chcederquist implemented the [aria-keyshortcuts](https://www.w3.org/TR/wai-aria-1.2/#aria-keyshortcuts) attribute. While well intended and - as far as I understand - implemented according to the spec, it creates some problems:

1. As stated in the spec itself, keys may be different for various keyboard layouts.
2. Currently there is no option in Leaflet to localize the keys in the language of the page. While this would be against the spec in it's current wording, it would help understanding (e.g. for users of screenreaders). As it is now, it's definitely a failure against [WCAG SC 3.1.2 Language of Parts](https://www.w3.org/WAI/WCAG22/Understanding/language-of-parts.html), when the language of the page is not English.

### Suggested solution

In my view the aria-keyshortcuts is not well defined and should be avoided at the moment. This is backed by https://github.com/w3c/aria/issues/2141 and the fact that other major maps don't use it (including Google Maps, Mapbox, MapTiler, Apple Maps, Bing Maps).

For some users of Leaflet it may be beneficial, however. That's why I suggest to make this feature optional. By default the attribute should not be set on the map container.

### Alternatives considered

- You could add options to localize the key names, but this would be complex and against the wording of the spec.
- Add an option to not use the attribute (=use by default). This would still produce failures against the WCAG in all languages except English.

**Comment by Falke-Design** [comment]
I'm not sure what the best is, both is valid. What do others think?
